src/django/api/templates/mail/approved_facility_claim_contributor_notice_body.txt

```diff
@@ -0,0 +1,12 @@
+Hi,
+
+You're receiving this email because we approved a facility claim for a facility
+appearing on one of your facility lists.
+
+The facility is:
+
+  - Facility: {{ facility_name }}, {{ facility_address }}
```

Comment:
Consider including the full facility country name in these notifications.

Reply:
Done in 0d3d6a29a1fa3bbfdafd95b0d042eaa5e1c58b2a